homeassistant/components/tuya/binary_sensor.py

```diff
@@ -261,11 +260,22 @@ class TuyaBinarySensorEntityDescription(BinarySensorEntityDescription):
     # https://developer.tuya.com/en/docs/iot/categorysos?id=Kaiuz3oi6agjy
     "sos": (
         TuyaBinarySensorEntityDescription(
-            key=DPCode.SOS_STATE,
+            key=DPCode.BASIC_INDICATOR,
             device_class=BinarySensorDeviceClass.SAFETY,
         ),
         TAMPER_BINARY_SENSOR,
     ),
+    # Smart Camera
+    # https://developer.tuya.com/en/docs/iot/s?id=K9gf48qapbl1n
+    "sp": (
+        TuyaBinarySensorEntityDescription(
+            key=DPCode.SOS_STATE,
```

Comment:
This doesn't seem correct according to Tuya specifications.
<https://developer.tuya.com/en/docs/iot/s?id=K9gf48qapbl1n>

Nor is it visible on the doorbells I have available.